python/hail/api2/matrixtable.py

```diff
@@ -2210,6 +2210,64 @@ def unpersist(self):
         """
         return MatrixTable(self._jvds.unpersist())
 
+    @handle_py4j
+    @typecheck_method(name=strlike)
+    def index_rows(self, name='row_idx'):
+        """Add the row integer index as a new field.
```

Comment:
I prefer "Add the integer index of each row as a new row field." to "row integer index".

Currently index on table says: "Add the numerical index of each row as a new field." and "The value of this column is the numerical index of each row".  Can you change numerical to integer there?